core/audits/network-requests.js

```diff
@@ -31,8 +31,8 @@ class NetworkRequests extends Audit {
   static async audit(artifacts, context) {
     const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
     const records = await NetworkRecords.request(devtoolsLog, context);
-    const earliestStartTime = records.reduce(
-      (min, record) => Math.min(min, record.startTime),
+    const earliestMainThreadStartTime = records.reduce(
```

Comment:
Can we stick to `earliestRendererStartTime` here, like the other PR has right now?

Reply:
whoops, must have been from the earlier naming round.